src/simulations/collect.jl

```diff
@@ -112,12 +113,13 @@ function run!(
     agent_step!,
     model_step!,
     n;
-    when = true,
-    when_model = when,
-    mdata = nothing,
-    adata = nothing,
-    obtainer = identity,
-    agents_first = true,
+    when=true,
```

Comment:
I don't understand these changes. It changes the more readable format of `x = y` to the less readable one of `x=y`. Regardless, even though this style can be debated of which one is better, such formatting style changes shouldn't be done in a PR that adds a new feature, but in an independent one.

Reply:
This was something Julia addon on VSCode changed automatically. I reverted them.